src/seedu/addressbook/parser/Parser.java

```diff
@@ -246,7 +237,9 @@ private Command prepareFind(String args) {
 
         // keywords delimited by whitespace
         final String[] keywords = matcher.group("keywords").split("\\s+");
-        final Set<String> keywordSet = new HashSet<>(Arrays.asList(keywords));
+        List<String> keywordsList = Arrays.asList(keywords);
+        keywordsList.replaceAll(String::toLowerCase);
```

Comment:
Every time you change the behavior of the application, you should update the user guide to reflect it.